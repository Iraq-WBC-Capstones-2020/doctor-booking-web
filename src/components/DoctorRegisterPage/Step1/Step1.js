import React from 'react'
import './style.css'
import { Form, Row, Col, Button, Container } from 'react-bootstrap';

function Step1() {

    return(
        <Container>

        <Form className="personal">
            <Row>
                <Col lg={6}>
                <Form.Group>
                <Form.Label name="Full name">
                    Full name
                    <Form.Control placeholder="Full name" />
                </Form.Label>
                </Form.Group>

                </Col>
                <Col lg={6}>
                <fieldset >

        <Form.Group>

        <Form.Label name="formHorizontalRadios">
        Gender
        
        <div className="myCheck mt-2">
        <Form.Check
          type="radio"
          label="female"
          name="formHorizontalRadios"
          id="formHorizontalRadios1"
        />
        <Form.Check
          type="radio"
          label="male"
          name="formHorizontalRadios"
          id="formHorizontalRadios2"
        />
        </div>
        </Form.Label>
       </Form.Group>

  </fieldset>
  </Col>
  </Row>    
  <Row>
                <Col lg={6}>
                <Form.Label name="E-mail">
                    Email
                    <Form.Control placeholder="Enter  your E-mail" />
                </Form.Label>
                </Col>
                <Col lg={6}>
                <Form.Label name="number">
                    Phone number
                    <Form.Control placeholder="Enter your phone number" />
                </Form.Label>
                </Col>
                </Row>

            <Row>       
                <Col lg={6}>
                    <Form.Label>Password
                        <Form.Control type="password" placeholder="Enter your password" />
                    </Form.Label>
                </Col>
                <Col lg={6}>
                    <Form.Label>Confirm password
                        <Form.Control type="password" placeholder="Enter your password again" />
                    </Form.Label>
                </Col>
            </Row> 
            
                <Row>
                <Col lg={12}>
                        <Form.File
                        className="position-relative"
                        required
                        name="file"
                        label="Personal photo"
                        
                        id="validationFormik107"
                        feedbackTooltip
                        />
                    </Col>
                    </Row>
               
        </Form>
        </Container>

    )
    
}

export default Step1