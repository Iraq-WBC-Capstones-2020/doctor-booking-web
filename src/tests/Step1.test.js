import React from 'react';
import { render } from '@testing-library/react';
import Step1 from '../components/DoctorRegisterPage/Step1/Step1';
import renderer from 'react-test-renderer';
it('Step 1 render1', () => {
  render(<Step1 />);
});

it('Step 1 renders correctly', () => {
  const tree = renderer.create(<Step1 />).toJSON();
  expect(tree).toMatchSnapshot();
});
