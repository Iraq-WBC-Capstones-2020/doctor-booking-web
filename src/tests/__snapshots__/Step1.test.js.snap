// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Step 1 renders correctly 1`] = `
<div
  className="container"
>
  <form
    className="personal"
  >
    <div
      className="formRow row"
    >
      <div
        className="col-lg-6"
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
            name="Full name"
          >
            Full name
            <input
              className="form-control"
              placeholder="Full name"
            />
          </label>
        </div>
      </div>
      <div
        className="col-lg-6"
      >
        <fieldset>
          <div
            className="form-group"
          >
            <label
              className="form-label"
              name="formHorizontalRadios"
            >
              Gender
              <div
                className="myCheck mt-2"
              >
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    disabled={false}
                    id="formHorizontalRadios1"
                    name="formHorizontalRadios"
                    type="radio"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="formHorizontalRadios1"
                    title=""
                  >
                    female
                  </label>
                </div>
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    disabled={false}
                    id="formHorizontalRadios2"
                    name="formHorizontalRadios"
                    type="radio"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="formHorizontalRadios2"
                    title=""
                  >
                    male
                  </label>
                </div>
              </div>
            </label>
          </div>
        </fieldset>
      </div>
    </div>
    <div
      className="formRow row"
    >
      <div
        className="col-lg-6"
      >
        <label
          className="form-label"
          name="E-mail"
        >
          Email
          <input
            className="form-control"
            placeholder="Enter  your E-mail"
            type="email"
          />
        </label>
      </div>
      <div
        className="col-lg-6"
      >
        <label
          className="form-label"
          name="number"
        >
          Phone number
          <input
            className="form-control"
            placeholder="ex: 0770-145-6788"
            type="tel"
          />
        </label>
      </div>
    </div>
    <div
      className="formRow row"
    >
      <div
        className="col-lg-6"
      >
        <label
          className="form-label"
        >
          Password
          <input
            className="form-control"
            placeholder="Enter your password"
            type="password"
          />
        </label>
      </div>
      <div
        className="col-lg-6"
      >
        <label
          className="form-label"
        >
          Confirm password
          <input
            className="form-control"
            placeholder="Enter your password again"
            type="password"
          />
        </label>
      </div>
    </div>
    <div
      className="formRow row"
    >
      <div
        className="col-lg-12"
      >
        <label
          name="photoUplod"
        >
          personal photo
        </label>
        <input
          className="btn btn-primary"
          disabled={false}
          id="photoUplod"
          type="button"
          value="uplode"
        />
         
      </div>
    </div>
  </form>
</div>
`;
